src/dsp_tools/models/sipi.py

```diff
@@ -3,25 +3,7 @@
 
 import requests
 
-from dsp_tools.models.exceptions import BaseError
-
-
-def on_api_error(res):
-    """
-    Checks for any API errors
-
-    Args:
-        res: the response from the API which is checked, usually in JSON format
-
-    Returns:
-        Knora Error that is being raised
-    """
-
-    if res.status_code != 200:
-        raise BaseError("SIPI-ERROR: status code=" + str(res.status_code) + "\nMessage:" + res.text)
-
-    if 'error' in res:
-        raise BaseError("SIPI-ERROR: API error: " + res.error)
+from dsp_tools.models.connection import check_for_api_error
```

Comment:
I thought it is not necessary that the `Sipi` class contains a duplicate of `check_for_api_error()`. It is the same logic, just that it's a SIPI error, not an API error.